configure.py

```diff
@@ -976,10 +981,16 @@ def configure_node(o):
   o['variables']['want_separate_host_toolset'] = int(cross_compiling)
 
   if not options.without_node_snapshot:
-    o['variables']['node_use_node_snapshot'] = b(not cross_compiling)
+    o['variables']['node_use_node_snapshot'] = b(not cross_compiling and not options.shared)
```

Comment:
Can you keep this <= 80 columns? You can wrap after e.g. the `b(`. Ditto on line 990.

Reply:
Fixed, thanks!